modules/monitoring-optimizing-alerting-rules-for-core-platform-monitoring.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+:_content-type: CONCEPT
+[id="Optimizing-alerting-rules-for-core-platform-monitoring{context}"]
+= Optimizing alerting rules for core platform monitoring
+
+Consider these key recommendations before creating or modifying alerting rules for core platform monitoring.
+By following these guidelines, you can optimize your alerting system, which helps you to build an efficient and effective monitoring environment tailored to your organization's specific needs
+
+* *Minimize the number of new rules*.
+Create only rules that are essential to your specific requirements.
+By minimizing the number of rules, you create a more manageable and focused alerting system in your monitoring environment.
+
+* *Focus on symptoms rather than causes*.
+Create rules that notify users of symptoms instead of underlying causes.
+This approach ensures that users are promptly notified of a relevant symptom so that they can investigate the root cause after an alert has triggered.
+This tactic also significantly reduces the overall number of rules you need to create.
+
+* *Plan and assess your needs before implementing changes*.
+First, decide what symptoms are important and what actions you want users to take if these symptoms occur.
+Then, assess existing rules and decide if you can modify any of them to meet your needs instead of creating entirely new rules for each symptom.
+By modifying existing rules and creating new ones judiciously, you help to streamline your alerting system.
+
+* *Provide clear alert messaging*.
+When you create alert messages, describe the symptom, the possible causes, and the recommended actions.
```

Comment:
Would this improve readability of this sentence?
```suggestion
When you create alert messages, describe the symptom, possible causes, and recommended actions.
```

Reply:
Yes. Changing it per your suggestion.